readme.md

```diff
@@ -50,14 +50,62 @@ app.on('ready', () => {
 });
 ```
 
-### Options
+## API
 
-#### showDevTools
+### Default exported function
+
+```js
+electronDebug({ showDevTools = false });
```

Comment:
no point in showing defaults here. that just looks confusing. same with the other places.